### PR TITLE
Add expiration to XML endpoints

### PR DESCRIPTION
**Pull request by KavajNaruj**

### CODE REVIEW

File: src/Response/XmlApiResponse.php [hunk 1]
```diff
@@ -18,18 +18,25 @@ class XmlApiResponse implements ApplicationIResponse
...
      * Create XmlApiResponse
      * This class only wrap JsonResponse from Nette and add possibility
      * to setup response code and automaticaly set content type
      *
      * @param integer $code
      * @param string $data
+     * @param string|int|bool|DateTimeInterface $expiration
      */
-    public function __construct($code, $data)
+    public function __construct($code, $data, $expiration = false)
```
Comment (lulco) [inline src/Response/XmlApiResponse.php:35]: I would prefer something consistent :) let's say DateTimeInterface|null and you can use it as typehint here

File: src/Response/XmlApiResponse.php [hunk 2]
```diff
@@ -48,7 +55,7 @@ public function getCode()
     public function send(IRequest $httpRequest, IResponse $httpResponse)
     {
         $httpResponse->setContentType('text/xml');
-        $httpResponse->setExpiration(false);
+        $httpResponse->setExpiration($this->expiration);
```
Comment (lulco) [inline src/Response/XmlApiResponse.php:58]: could this be null or have to be false in this version?

Comment (lulco) [reply src/Response/XmlApiResponse.php:58]: if false, than use $this->expiration ?: false

File: src/Response/XmlApiResponse.php (hunk 1 above)
Comment (lulco) [reply src/Response/XmlApiResponse.php:35]: ok php version is 5.6 :) I take it back :)

Comment (KavajNaruj) [reply src/Response/XmlApiResponse.php:35]: this version si for php 5.6, there is no type hint for null :(

File: src/Response/XmlApiResponse.php (hunk 2 above)
Comment (KavajNaruj) [reply src/Response/XmlApiResponse.php:58]: so this will not have a null value

File: src/Response/XmlApiResponse.php [hunk 3]
```diff
@@ -18,18 +19,25 @@ class XmlApiResponse implements ApplicationIResponse
      */
     private $response = null;
 
+    /**
+     * @var string|int|bool|DateTimeInterface
```
Comment (tomaj) [inline src/Response/XmlApiResponse.php:23]: isn't this so wide? why not keep one of them and be more strict? I know that all of those can be used in `IResponse->setExpiration` but I personally don't like it to have too open

Comment (lulco) [reply src/Response/XmlApiResponse.php]: Changed to DateTimeInterface|null, could we merge?

**Pull request merged**